pillar/vector/mitx.sls

```diff
@@ -530,3 +531,17 @@ vector:
         type: elasticsearch
         endpoint: 'http://operations-elasticsearch.query.consul:9200'
         index: logs-authlog-%Y.%W
+
+      s3_tracking:
+        inputs:
+          - tracking_log_timestamp_renamer
+        type: aws_s3
+        bucket: odl-residential-tracking-data
```

Comment:
The logs are currently uploaded to a `/logs` subdirectory.

Reply:
:+1: